app/src/main/java/com/asfoundation/wallet/promotions/PromotionsFragment.kt

```diff
@@ -60,32 +67,41 @@ class PromotionsFragment : DaggerFragment(), PromotionsView {
 
   override fun setLevelIcons() {
     for (i in 0..4) {
-      gamification_progress_bar.setLevelIcons(i)
+      legacy_gamification_progress_bar.setLevelIcons(i)
     }
   }
 
   override fun setStaringLevel(userStatus: UserRewardsStatus) {
```

Comment:
We should set the name of the methods to reflect that we are dealing with the legacy layout